### DISCUSSION TITLE
How to select a range of one day with the DateRangePicker

### DISCUSSION THREAD

**Discussion by cesalberca** [mantinedev/mantine#1190] [Q&A] [answered]
Is there a way to allow the users to select a range of only one day using the DateRangePicker?

**Comment by rtivital** [comment, marked as answer]
`<DateRangePicker allowSingleDateInRange />`

**Reply by cesalberca** [reply]
Yes! Thank you! I don't know how I could miss that, as it is well documented here: https://mantine.dev/dates/date-picker/?t=props
